app/src/main/java/org/inspire/breath/data/Volunteer.java

```diff
@@ -0,0 +1,44 @@
+package org.inspire.breath.data;
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+import android.support.annotation.NonNull;
+
+@Entity(tableName = "volunteer_table")
+public class Volunteer {
```

Comment:
Not sure if we need a volunteer table?

Reply:
Mainly put this in as an example - I would suggest keeping this for now and modifying it to be the "users" table down the line

Reply:
Should change it to a patient table now so rather than having people be confused by the appearance of a volunteer db.

Reply:
Changed it to a patient table as suggested, currently only has basic info columns (id, name, age). I guess we'll add columns for logged info etc. later as needed? Just let me know if you want me to add more before merging.